glue/cirq/stimcirq/_cirq_to_stim.py

```diff
@@ -5,15 +5,13 @@
 
 import cirq
 import stim
-import warnings
-
-_STIMCIRQ_TAG_LOOKUP = {"H": cirq.H, "X": cirq.X, "Y": cirq.Y, "Z": cirq.Z}
 
 
 def cirq_circuit_to_stim_circuit(
     circuit: cirq.AbstractCircuit,
     *,
     qubit_to_index_dict: Optional[Dict[cirq.Qid, int]] = None,
+    custom_op_conversion_func: Callable | None = None
```

Comment:
Expand the Callable type to clarify the function's intent.

Also add it to the docstring.